new/LIT6288Prayer.xml

```diff
@@ -0,0 +1,73 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6288Prayer" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ናስተበቍዓ፡ ለእግዝእተ፡ ኵልነ፡ ማርያም፡ እምነ፡ ወእሙ፡ ለእግዚእነ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Saʾali lana Māryām ʾǝmmǝna waʾǝmmu laʾƎgziʾǝna</title>
```

Comment:
Is there a transliteration error here?

Reply:
ʾǝmmǝna is "our mother" (እም+ነ፡) not "from us". Perhaps laʾƎgziʾǝna might be rendered laʾƎgziʾna, I do not know BM policy about this particular case.